util/rate_limiter.cc

```diff
@@ -100,17 +101,21 @@ void GenericRateLimiter::SetBytesPerSecond(int64_t bytes_per_second) {
 
 void GenericRateLimiter::Request(int64_t bytes, const Env::IOPriority pri,
                                  Statistics* stats) {
-  assert(bytes <= refill_bytes_per_period_.load(std::memory_order_relaxed));
   TEST_SYNC_POINT("GenericRateLimiter::Request");
   TEST_SYNC_POINT_CALLBACK("GenericRateLimiter::Request:1",
                            &rate_bytes_per_sec_);
+  if (pri == Env::IO_HIGH) {
+    total_bytes_through_[Env::IO_HIGH] += bytes;
```

Comment:
looks like `total_bytes_through_` is recorded but not used anywhere. what it is used for?

Reply:
`GetTotalBytesThrough`, used in unit test.